staging/src/k8s.io/client-go/tools/clientcmd/overrides.go

```diff
@@ -155,8 +158,9 @@ func RecommendedConfigOverrideFlags(prefix string) ConfigOverrideFlags {
 		ClusterOverrideFlags: RecommendedClusterOverrideFlags(prefix),
 		ContextOverrideFlags: RecommendedContextOverrideFlags(prefix),
 
-		CurrentContext: FlagInfo{prefix + FlagContext, "", "", "The name of the kubeconfig context to use"},
-		Timeout:        FlagInfo{prefix + FlagTimeout, "", "0", "The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests."},
+		CurrentContext:       FlagInfo{prefix + FlagContext, "", "", "The name of the kubeconfig context to use"},
+		Timeout:              FlagInfo{prefix + FlagTimeout, "", "0", "The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests."},
+		IncludeUninitialized: FlagInfo{prefix + FlagIncludeUninitialized, "", "false", "If true, will include the objects which have a non-empty initializer list"},
```

Comment:
Also question: 

I see the default value is `false`. Does this mean `kubectl get pods --include-uninitialized` still means includeUnintialized=false? Should the user explicitly set `=true` or does this work like `--all-namespaces` already with this code?